go/chat/sync.go

```diff
@@ -273,24 +273,43 @@ func (s *Syncer) getShouldUnboxSyncConvMap(ctx context.Context, convs []chat1.Co
 		if m[conv.GetConvID().String()] {
 			continue
 		}
-		if conv.Metadata.Status == chat1.ConversationStatus_BLOCKED {
-			continue
-		}
-		switch conv.GetMembersType() {
-		case chat1.ConversationMembersType_TEAM:
-			// include if this is a simple team, or the topic name has changed
-			if conv.GetTopicType() != chat1.TopicType_CHAT ||
-				conv.Metadata.TeamType != chat1.TeamType_COMPLEX ||
-				conv.GetConvID().Eq(s.GetSelectedConversation()) {
-				m[conv.GetConvID().String()] = true
-			}
-		default:
+		if s.shouldUnboxSyncConv(conv) {
 			m[conv.GetConvID().String()] = true
 		}
 	}
 	return m
 }
 
+func (s *Syncer) shouldUnboxSyncConv(conv chat1.Conversation) bool {
+	// Skips convs we don't care for.
+	switch conv.Metadata.Status {
+	case chat1.ConversationStatus_BLOCKED,
```

Comment:
I'd drop `MUTED` from here, otherwise the snippet will be old

Reply:
ah right